NOTICE

```diff
@@ -0,0 +1,2123 @@
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties.
+Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com,
+or you may send a check or money order for US $5.00, including the product name,
+the open source component name, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the extent
+required to debug changes to any libraries licensed under the GNU Lesser General Public License.
+
+
+-------------------------------------------------------------------
+
+ms-iot/edk2 8594c2073cdb1065e60f01e9b099918c5e839212 - BSD-2-Clause-Patent
+(c) 2007
+Copyright (c) 1999
+Copyright (c) 2006
+Copyright (c) 2007
+Copyright (c) 2016
+Copyright 2018 NXP
+(c) 2011, Apple Inc.
+Copyright PreMap Map
+Oniguruma (c) K.Kosako
+Copyright (c) 1999 - 20
+Copyright (c) 2002-2018
+Copyright (c) 1999 - 201
+Copyright (c) 2007 - 201
+Copyright (c) 2011, ARM.
+Copyright (c) 2013, ARM.
+Copyright 2015 Dell Inc.
+Copyright 2016 Dell Inc.
+Copyright (c) 2007 - 2011
+Copyright (c) 2008 - 2011
+Copyright 2014 Apple Inc.
+Copyright (c) 2015 ARM Ltd
+Copyright 2010 Google Inc.
+Copyright 2013 Google Inc.
+Copyright 2014 Google Inc.
+Copyright 2015 Google Inc.
+Copyright 2016 Google Inc.
+Copyright 2017 Google Inc.
+Copyright 2018 Google Inc.
+Coprocessor Status Register
+Coprocessor Trap Reg (CPTR)
+Copyright (c) 2017, Linaro.
+Copyright (c) 2018, Linaro.
+Copyright (c) 2011, ARM Ltd.
+Copyright (c) 2012, ARM Ltd.
+Copyright (c) 2013, ARM Ltd.
+Copyright (c) 2014, ARM Ltd.
+Copyright (c) 2015, ARM Ltd.
+Copyright (c) 2017, AMD Inc.
+Copyright (c) 2017, ARM Ltd.
+Copyright (c) 2018, ARM Ltd.
+Copyright (c) 2019, ARM Ltd.
+Copyright (c) 2017, AMD, Inc.
+Copyright (c) 2004, K A Fraser
+Copyright (c) 2008 Doug Rabson
+Copyright (c) 2008, Apple Inc.
+Copyright (c) 2009, Apple Inc.
+Copyright (c) 2010, Apple Inc.
+Copyright (c) 2011, Apple Inc.
+Copyright (c) 2012, Apple Inc.
+Copyright (c) 2015 Linaro Ltd.
+Copyright (c) 2016 Linaro Ltd.
+copyright (c) 2011, Apple Inc.
+Copyright (c) 2005, Keir Fraser
+Copyright (c) 2010, Apple, Inc.
+Copyright (c) 2011, Apple, Inc.
+Copyright (c) 2014, Citrix Ltd.
+Copyright (c) 2014, Linaro Ltd.
+Copyright (c) 2015, Linaro Ltd.
+Copyright (c) 2016, Linaro Ltd.
+Copyright (c) 2017, Linaro Ltd.
+Copyright (c) 2018, Linaro Ltd.
+Copyright (c) 2019, Linaro Ltd.
+Copyright DPCopyright CheckDict
+Copyright', FileTable, ID Check
+Coprocessor Trap Register (CPTR)

```

Comment:
Weird output?

Reply:
CELA should be reviewing these once they are done. I'll highlight this for them.

Definately looks like a fuzzy match gone wrong though.